cmd/ttn-lw-cli/commands/applications.go

```diff
@@ -252,12 +252,15 @@ func init() {
 	applicationsListCommand.Flags().AddFlagSet(selectApplicationFlags)
 	applicationsListCommand.Flags().AddFlagSet(paginationFlags())
 	applicationsListCommand.Flags().AddFlagSet(orderFlags())
+	applicationsListCommand.Flags().Bool("all", false, "Show all application fields")
```

Comment:
Consider making `all` part of `selectXXXFlags`

Reply:
`util.SelectFieldMask` uses `selectXXXFlags` to filter arguments. If we make `all` part of it, then we also have to update `util.SelectFieldMask` to discard it in the result value. I think this would cause more harm than good.

Reply:
In that case, add an `allFlag`. My comment was most to avoid repetition than making it part of `selectXXXFlags()`